### DISCUSSION TITLE
重复付费请退款

### DISCUSSION THREAD

**Discussion by Git2606** [getlantern/lantern-forum-cn#464] [Q&A]
账号：1592273489@qq.com

2024年4月18日会员续费，但是输入信用卡信息后，蓝灯APP上提示未支付成功，网络卡顿， 导致多次输入信用卡信息，被多次扣费，我只需要续1年的费。请将2年的付款退费！

![20240423144504](https://github.com/getlantern/lantern-forum-cn/assets/18511209/36fb771b-f46d-47f6-a7a2-b3aeef5262b3)
![20240423144513](https://github.com/getlantern/lantern-forum-cn/assets/18511209/8875bc70-fd0a-47bb-871b-2d0ddcc67ea6)

**Comment by MaxMKMax** [comment]
@Git2606 您好! 好的, 我会帮您申请2年的错误支付的退款. 我们可以帮您手动升级专业版. 非常抱歉影响了您的购买体验, 谢谢您支持蓝灯!

也请问一下您的1年专业版有被升级成功吗?

**Reply by Git2606** [reply]
感谢提供的帮助! 是升级成功的，现在显示有效期到2027/08/02。帮我减去2年时间，退费2年就好。

**Reply by MaxMKMax** [reply]
@Git2606 您好. 我已经帮您提交错误付款的退款申请了. 请您等下周付款团队帮您进行退款. 谢谢.

**Reply by MaxMKMax** [reply]
@Git2606 已经做好错误付费的退款了. 请等待系统完成. 谢谢!

**Comment by Git2606** [comment]
帮忙取消两年的付款，保留一年的，谢谢



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月24日(周三) 下午5:02
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 重复付费请退款 (Discussion #464)




 
@Git2606 您好! 好的, 我会帮您申请错误支付的退款. 但在这之前, 我想请问您要保留一年的年费吗? 我们可以帮您手动升级专业版. 非常抱歉影响了您的购买体验, 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@Git2606 嗯嗯好的. 上周已经提交上去了. 请您等待. 谢谢啊!

**Reply by 1etmein** [reply]
@Mike777Khoo 
我的情况和他类似，详细的发你邮箱了。收到请回复下，谢谢。帐号rwtgb@live.com

**Reply by MaxMKMax** [reply]
@1etmein 您好. 您发邮件给我的是这个邮箱吗? 请您也提供我付款单据. 谢谢哦!

**Reply by 1etmein** [reply]
@Mike777Khoo 
付款单据见邮箱附件

**Reply by 1etmein** [reply]
![Screenshot_2024-05-04-09-38-40-511_com tencent mm](https://github.com/getlantern/lantern-forum-cn/assets/22019619/a8837e8a-261a-49f2-bf44-c83f2fd9e4c7)

**Comment by MaxMKMax** [comment]
@Git2606 好的. 前几天已经帮您提交上去了. 等待下周付款团队帮您处理!

**Reply by 1etmein** [reply]
你好，请问我的处理结果怎么样了。
我是1etmein    rwtgb@live.com。
相关资料都发到你的邮箱了。
是不是回复错人了。

**Reply by 1etmein** [reply]
@Mike777Khoo 
你好，请问我的处理结果怎么样了。
我的情况和GIT2606相似。但我不是他，
我是1etmein    邮箱rwtgb@live.com。
希望你没有搞混我和他。
相关资料都发到你的邮箱了，你也可以在过往对话记录找到。
如果有明确结果，请在这里或者邮件回复下，谢谢。

**Reply by MaxMKMax** [reply]
@1etmein 好的. 我帮您查看. 谢谢您通知我.

**Reply by MaxMKMax** [reply]
@1etmein 
您好. 已经给您弄好错误重复的退款了. 请您等待 7-10 天系统完成退款. 谢谢.
